README.md

```diff
@@ -155,6 +157,9 @@ jobs:
       - name: Create Release Pull Request or Publish to npm
         id: changesets
         uses: changesets/action@v1
+        with:
+          # Include this to create GitHub releases and Git tags
+          # publish: yarn changeset tag
```

Comment:
tbf - I think I have successfully used Changesets with custom release scripts like this:
```yml
      - name: Create Release Pull Request or Publish to npm
        id: changesets
        uses: changesets/action@v1
        with:
          publish: yarn run release
```

where the `release` script be defined like this:
```json
    "release": "node ./scripts/release.js && yarn changeset tag"
```

This way you don't have to rely on `steps.changesets.outputs.hasChangeset` at all